### PR TITLE
Home-about page

### PR DESCRIPTION
**Pull request by j-tee**
[Link to video](https://www.loom.com/share/58b5e2a264e346ae92ee5ea0e3e62eeb
**Build** only these 2 pages:)

- The home page.

- The about page.

- [ ] Each of these pages should have versions for 2 different screen sizes:

- [ ] Mobile: up to 768px wide.

- [ ] Desktop: 768px or wider.

- Interactions

Links

- [ ] The home page should have a link in the menu to the about page.

- [ ] 

- [ ] The logo in the header links to the home page.

- Mobile menu

- [ ] When the user clicks (or taps) the hamburger button on the header, the mobile menu appears over the page.

- [ ] The mobile menu should have a close (X) button that closes the menu.

- Dynamic page

- [ ] The section "Featured speakers" should be created dynamically in JavaScript.

- [ ] Use a JavaScript variable with the data about the speakers and use it when the page loads to create the HTML for this section dynamically.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+    <hr class="lowerBar">
+
+    <div id="stafflist">
+
+    </div>
+    <button type="button" class="more btn btn-outline-dark btn-lg">More <span class="">&#8744;</span>
+    </button>
+  </section>
+  <section id="section-4">
+    <h3>Partners</h4>
```
Comment (awais-amjed) [inline index.html]: Kindly fix the typo here, the opening and closing tags should be the same, either `h3` or `h4` 🤗

File: index.html [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+      <h2 class="c-hours">CLEANING HOURS - 9:00 AM TO 5:00 PM (MON - FRI)</h2>
+      <p class="c-msg">We have the best offers available on the market</p>
+    </div>
+  </section>
+  <section id="section-2">
+    <h4>Our Services</h4>
+    <hr class="lowerBar">
+    <div class="services">
+      <div class="service-list">
+        <img src="./assets/images/washing-machine-nogb.png" alt="" />
```
Comment (awais-amjed) [inline index.html]: Kindly add a meaningful `alt` text to this `img` tag 🤗

File: index.html [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+        <h4>Laundry Services</h4>
+        <p>
+          We undertake laundry services for both Domestic and commercial
+          facilities
+        </p>
+      </div>
+      <div class="service-list">
+        <img src="./assets/images/householdcores-nobg.png" alt="" />
```
Comment (awais-amjed) [inline index.html]: Kindly add a meaningful `alt` text to this `img` tag 🤗

File: index.html [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+        <h4>Household Chores</h4>
+        <p>
+          Tired of domestic household chores or simply need a helping! We've got you covered
+        </p>
+      </div>
+      <div class="service-list">
+        <img src="./assets/images/swimmingpool-nobg.png" alt="" />
```
Comment (awais-amjed) [inline index.html]: Kindly add a meaningful `alt` text to this `img` tag 🤗

File: index.html [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+        <h4>Swimming Pool Cleaning</h4>
+        <p>
+          Keep your swimming clean and hygienic. We provide industry leading pool solutions
+        </p>
+      </div>
+      <div class="service-list">
+        <img src="./assets/images/landscaping-nobg.png" alt="" />
```
Comment (awais-amjed) [inline index.html]: Kindly add a meaningful `alt` text to this `img` tag 🤗

File: index.html [hunk 6]
```diff
@@ -0,0 +1,144 @@
...
+        <h4>Landscaping</h4>
+        <p>
+          Indulge yourself! Lets us provide you with a captivating and beautiful scenery every time you step out into your compund
+        </p>
+      </div>
+      <div class="service-list">
+        <img src="./assets/images/handyman.svg" alt="" />
```
Comment (awais-amjed) [inline index.html]: Kindly add a meaningful `alt` text to this `img` tag 🤗

File: index.html [hunk 7]
```diff
@@ -0,0 +1,144 @@
...
+  </section>
+  <section id="section-4">
+    <h3>Partners</h4>
+      <hr class="lowerBar">
+      <ul class="partners">
+        <li><img class="abt-image" src="./assets/images/mozilla.png" alt=""></li>
+        <li><img class="abt-image" src="./assets/images/googlekorea-removebg-preview.png" alt=""></li>
+        <li><img class="abt-image" src="./assets/images/naver-removebg-preview.png" alt=""></li>
+        <li><img class="abt-image" src="./assets/images/kakao-removebg-preview.png" alt=""></li>
+        <li><img class="abt-image" src="./assets/images/airbnb-removebg-preview.png" alt=""></li>
```
Comment (awais-amjed) [inline index.html]: Kindly add a meaningful `alt` text to all the `img` tags 🤗

File: about.html [hunk 8]
```diff
@@ -0,0 +1,129 @@
...
+    <section id="abt-section-2">
+      <h3>Hi-tech Quality Cleaning and Repairworks Services</h3>
+      <hr class="lowerBar">
+      <p>
+        Lorem ipsum dolor sit amet consectetur, adipisicing elit. Amet
+        temporibus placeat atque qui fuga autem quibusdam iste hic, dolorem
+        Lorem ipsum dolor sit amet consectetur, adipisicing elit. Amet
+      </p>
+      <div class="abt-logo">
+        <img src="./assets/images/logo.svg" alt="" />
```
Comment (awais-amjed) [inline about.html]: Kindly add a meaningful `alt` text to this `img` tag 🤗

File: about.html [hunk 9]
```diff
@@ -0,0 +1,129 @@
...
+    </section>
+    <section id="abt-section-4">
+      <h4>Partners</h4>
+      <hr class="lowerBar">
+      <ul class="partners">
+        <li><img class="abt-image" src="./assets/images/mozilla.png" alt=""></li>
+        <li><img class="abt-image" src="./assets/images/googlekorea-removebg-preview.png" alt=""></li>
+        <li><img class="abt-image" src="./assets/images/naver-removebg-preview.png" alt=""></li>
+        <li><img class="abt-image" src="./assets/images/kakao-removebg-preview.png" alt=""></li>
+        <li><img class="abt-image" src="./assets/images/airbnb-removebg-preview.png" alt=""></li>
```
Comment (awais-amjed) [inline about.html]: Kindly add a meaningful `alt` text to all these `img` tags 🤗

File: index.html [hunk 10]
```diff
@@ -0,0 +1,144 @@
...
+      <p>
+        Lorem, ipsum dolor sit amet consectetur adipisicing elit. Consequatur,
+        dolores ipsa quo sunt omnis nemo autem corrupti minima sapiente
+        veritatis rem culpa consectetur cupiditate vitae amet aliquam non
+        necessitatibus iste.
+      </p>
+      <h2 class="c-hours">CLEANING HOURS - 9:00 AM TO 5:00 PM (MON - FRI)</h2>
+      <p class="c-msg">We have the best offers available on the market</p>
+    </div>
+  </section>
```
Comment (awais-amjed) [inline index.html:50]: Kindly fix the spacing around elements and introduce some space at the top of this section to resemble the given design in the mobile version 🤗

Your design:

image

Template design:

image

**Review by awais-amjed — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️

Hi @j-tee , 

Good job so far! 👏👏
There are some issues that you still need to work on to go to the next project but you are almost there! 🥳🙌🤗

## Highlights

- [x] Descriptive Pull Request ✔️
- [x] No Linter issues ✔️ 
- [x] Awesome design Design ✔️
- [x] Good code organization ✔️ 

gif

## Required Changes ♻️

- [ ] I saw your loom video presentation, it was awesome 👏👏 but you forgot to add it in the PR description as described in the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/capstone/html_capstone.md#project-documentation), Kindly add that 🤗
image
- [ ] Kindly give appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/) in the acknowledgment section.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by franclobo — APPROVED** [review]
Hi @j-tee, 

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**